pkgs/misc/uboot/default.nix

```diff
@@ -19,10 +20,10 @@
 }:
 
 let
-  defaultVersion = "2021.10";
+  defaultVersion = "2022.01";
```

Comment:
Why not the newer version?

Reply:
Because our precious patches don't work there yet.

I might try rebasing them later, but I must admit I'm running out of gas :P

Reply:
[See other comment too]

I know I haven't been updating the last two releases (I wonder why r-ryantm didn't notice, and no one else did), but the goal is to prevent any kind of foward-incompatible drift.